### ISSUE TITLE
Issue with Future Proofing api pulls

### ISSUE THREAD

**Issue by XinnyLiuu** [liujustin/CoinIt#13]
Images are currently set to load when we match them with a currency's name.
However if a new currency is added, then the image wont load because we dont have the image on file for the respective currency

**Comment by liujustin** [comment]
Yeah we need to grab most/all of the outstanding cryptocurrency logos

**Comment by liujustin** [comment]
looking into this https://github.com/cjdowner/cryptocurrency-icons
